src/canvas.jl

```diff
@@ -229,3 +229,48 @@ function annotate!(
     char_point!(c, char_x, char_y, text, color)
     c
 end
+
+function printcolorbarrow(
+    io::IO, c::Canvas, row::Int, colormap::Any, border::Symbol,
```

Comment:
Arbitrary `Canvas`, for https://github.com/JuliaPlots/UnicodePlots.jl/issues/169.